### PR TITLE
chore(css): ensure the menu is wide enough, but no !important here

### PR DESCRIPTION
**Pull request by redox**

### CODE REVIEW

**Comment by pixelastic** [comment]
Did you check on small mobile devices?

**Comment by redox** [comment]
Arg no

**Comment by redox** [comment]
@pixelastic safer like that

File: src/styles/main.scss [hunk 1]
```diff
@@ -111,6 +109,9 @@
 // BREAKPOINT 1:
 // Screen is big enough to display the text snippets
 @media (min-width: $breakpoint-medium) {
+  .aa-dropdown-menu {
+    min-width: 400px;
```
Comment (pixelastic) [inline src/styles/main.scss]: Could you use the `$breakpoint` variables instead of hardcoding the width?

Comment (redox) [inline src/styles/main.scss]: Hmmm, something like `$breakpoint-medium * 0.75` ?

Comment (pixelastic) [inline src/styles/main.scss]: Yes maybe, or just putting 400px in a variable defined at the top so it feels a bit less magic (and overridable by users).

Comment (redox) [inline src/styles/main.scss]: Go for the variable then.

**Pull request merged**